src/isIsogram.test.js

```diff
@@ -8,12 +8,24 @@ describe(`Function 'isIsogram':`, () => {
   });
 
   it(`should return a boolean`, () => {
-
+    expect(typeof isIsogram('playgrounds')).toBe('boolean');
   });
 
   it(`should be case sensitive`, () => {
+    const result = isIsogram('wOod');
+
+    expect(result).toBe(false);
+  });
 
+  it(`should be return true if word is empty`, () => {
+    const result = isIsogram('');
+
+    expect(result).toBe(true);
   });
 
-  // write more tests here
+  it(`should be return true if word is empty`, () => {
```

Comment:
Rename this test. For example: `input: 'forest' should return true`

Reply:
I agree that you have to rename it. It can be `should be return true if word is an isogram`